server/src/cloud/board.ts

```diff
@@ -366,4 +383,37 @@ export const initializeBoardFunctions = () => {
     await removeFromModerators(Parse.User.createWithoutData(request.userId), request.boardId);
     return {status: "Success", description: "User was successfully removed from the list of moderators"};
   });
+
+  /**
+   * Cancel voting
+   */
+  api<{boardId: string}, {status: string; description: string}>("cancelVoting", async (user, request) => {
```

Comment:
What will happend if two moderators press the cancel button at the exact same time? Won't it delete the current and previous voting data? 

Maybe you should add a additional check that the voting can only be cancelled (and deleted) if it's active.

Reply:
Good point. I added an additional check.